src/UXClient/Components/LineChart/LineChart.ts

```diff
@@ -33,6 +33,8 @@ class LineChart extends TemporalXAxisComponent {
     private hasStackedButton: boolean = false;
     private stackedButton: any = null;
     private visibleAggCount: number;
+    private swimLaneLabels: any;
```

Comment:
nit: can we define an interface for swimLaneLabels instead of any? Something like this: https://stackoverflow.com/a/36467564

Reply:
^ That ```private swimLaneLabels: any;```  is actually a D3 selection with a REALLY bad name.  haha.  I'll work on an interface for the actualy swimLaneLabels map and update that name to something more explanatory.